src/components/Todo.tsx

```diff
@@ -8,20 +10,25 @@ interface Todo {
 }
 
 const Todo = () => {
-  const [todo, setTodo] = useState<string>("");
   const [id, setId] = useState(0);
+  const [todo, setTodo] = useState<string>("");
   const [todoList, setTodoList] = useState<Todo[]>([]);
 
+  const { response, error, loader } = useFetch(TODO_ENDPOINT);
+
+  useEffect(() => {
+    !error ? setTodoList(response) : console.log(error);
```

Comment:
No need of condition here. You can simply setTodoList here
& below in JSX you can use conditional rendering to show error